dev/ci/ci-coq_dpdgraph.sh

```diff
@@ -10,8 +10,6 @@ git_download coq_dpdgraph
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/coq_dpdgraph"
-  autoconf
-  ./configure
-  make
-  make test-suite
+  dune build @install -p coq-dpdgraph
```

Comment:
Adding 
```suggestion
  dune build @install -p coq-dpdgraph @runtest 
```
like in the opam file fails so I don't know how to run the tests.

cc @ybertot @palmskog

Reply:
What is the failure?

Reply:
have a look